apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/component/config/PortalConfig.java

```diff
@@ -242,6 +242,14 @@ public String consumerTokenSalt() {
     return getValue("consumer.token.salt", "apollo-portal");
   }
 
+  public int openApiLimitCount() {
+    return getIntProperty("open.api.limit.count", 20);
+  }
+
+  public boolean isOpenApiLimitEnabled() {
```

Comment:
Since rate limits are set at the token level, adding a global flag seems unnecessary. Instead, we could add an "Enable Rate Limit" option on the consumer token creation page. Users can toggle this option to configure the rate limit if desired; otherwise, leaving it unchecked will set the rate limit to 0.